modules/features/shared/src/main/java/au/com/shiftyjelly/pocketcasts/shared/AppLifecycleObserver.kt

```diff
@@ -69,6 +73,7 @@ class AppLifecycleObserver(
         applicationScope.launch {
             notificationScheduler.setupReEngagementNotification()
             notificationScheduler.setupTrendingAndRecommendationsNotifications()
+            blazeAdsManager.updateAds()
```

Comment:
ℹ️ This might not be enough. It will only run once but our app can be alive for a long time since the playback service will keep it alive. Id's suggest either running it with every `onStart()` or inside a `CouroutineWorker`.